### DISCUSSION TITLE
Many conventional P25 channels with TGID=1

### DISCUSSION THREAD

**Discussion by n1zyy** [chuot/rdio-scanner#402] [Q&A]
Howdy!

I'm running sdrtrunk and feeding it to an rdio-scanner Docker container. I'm monitoring a number of conventional P25 channels, almost all of which have a talkgroup ID of 1.

I'm wondering what the best practice is for keeping these separate. I'm still trying to make sense of exactly what is happening, but I have a hunch that within an rdio-scanner system, talkgroup is presumed to be a unique identifier. The display on playback seems to use the correct label, but in the Select TG and Search Call views, it seems like the label is whatever most recently used that TGID.

How are people working around this? With analog channels, sdrtrunk lets me assign an arbitrary talkgroup, which I'm able to make unique, and with trunked systems each trunked system has its own rdio-scanner System. I suspect the simplest thing would be to create a unique System per conventional channel, but this feels a little clunky. It's possible the solution is actually a more clever setup in sdrtrunk? I'm fairly new to all of this.

**Comment by Thinline-Dynamic-Solutions** [comment]
You can create one Convention System and then assign talkgroup id’s in an
alias list. Then type the talkgroup id into the channel page that you want
each channel to use.

*Michael Chambers - Ohio Radio Scanner Network*
Founder/Manager/Support
***@***.**** ***@***.***>

**Comment by CKrajnak** [comment]
Hey there. Were you ever able to resolve this issue? I'm running into the same problem as well and I cannot find an answer anywhere else

**Comment by Thinline-Dynamic-Solutions** [comment]
My only other recommendation would be to put each one on its own system.
Essentially this isn’t a software bug, a trunking system is meant to not
have duplicate TGID’S. If you talkgroups have to be #1 then you would have
to put them each in their own separate system.
